lib/linguist/languages.yml

```diff
@@ -6784,6 +6784,21 @@ V:
   codemirror_mode: go
   codemirror_mime_type: text/x-go
   language_id: 603371597
+VB6:
+  type: programming
+  color: "#2c6353"
+  extensions:
+  - ".ctl"
+  - ".dsr"
```

Comment:
This extension isn't popular enough for inclusion.

```suggestion
```

Please also remove the `.dsr` sample.

Reply:
Actually, the version with the first letter in upper case is more common in the wild (`.Dsr`). I used the lower case version since that's what [Microsoft documentation](https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-basic-6/aa241721(v=vs.60)) seems to suggest, but I can make the change.